### ISSUE TITLE
package doesnt match referenced package in fabric.mod.json

### ISSUE THREAD

**Issue by Linguardium** [redcreeper14385/fabric-for-fabric#5]
https://github.com/redcreeper14385/fabric-for-fabric/blob/e54e6ecba26c7906c82c650613e7cfc0ffc9435c/src/main/java/mounderfod/fabricforfabric/FabricForFabric.java#L1

https://github.com/redcreeper14385/fabric-for-fabric/blob/e54e6ecba26c7906c82c650613e7cfc0ffc9435c/src/main/resources/fabric.mod.json#L22

causes crash on load 
```
Caused by: net.fabricmc.loader.api.LanguageAdapterException: java.lang.ClassNotFoundException: mounderfod.fabricforfabric.FabricForFabric
	at net.fabricmc.loader.util.DefaultLanguageAdapter.create(DefaultLanguageAdapter.java:47)
```

**Comment by redcreeper14385** [comment]
@Linguardium Will fix in next update, for now please use version 1.1.0 instead.

**Comment by Linguardium** [comment]
fixed in 5532fb4
